JSBadgeView/JSBadgeView.h

```diff
@@ -39,6 +39,13 @@ typedef NS_ENUM(NSUInteger, JSBadgeViewAlignment)
 
 @property (nonatomic, copy) NSString *badgeText;
 
+@property (nonatomic, assign) NSUInteger badgeNumber;
```

Comment:
This introduces 2 ways of changing the contents of the badge: one with a string, one with a number. I think that's a bit confusing.